app/views/landmarks/index.erb

```diff
@@ -0,0 +1,14 @@
+<table>
+  <th>Name</th>
+  <th>Year Completed</th>
+  <tr>
+    <% @landmark.each do |land| %>
```

Comment:
also `land` as a block variable doesn't make sense. It's not all of a sudden a piece of land right? isn't is `landmark`?